src/components/material/MaterialHeader.tsx

```diff
@@ -42,8 +45,12 @@ const MaterialHeader: React.FC<MaterialHeaderProps> = ({
     workId: wid
   },
   manifestation,
-  selectManifestationHandler
+  selectManifestationHandler,
+  selectPeriodikumSelect
 }) => {
+  // THIS MUST BE DELETED (START
+  const isDemoPeriodikum = getUrlQueryParam("periodikum");
+  // THIS MUST BE DELETED (END)
```

Comment:
Please prefix such comments with `TODO` or remove this feature entirely since we have already demoed this.

The convention of using `TODO` makes it easier for us to search through the codebase for issues that needs to be addressed in the future.